### PR TITLE
Set array item types to string

### PR DESCRIPTION
**Pull request by felixbarny**
relates to #641

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -44,6 +44,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485]
 - Fix panic when a signal is delivered before the server is instantiated {pull}580[580]
 - Use type integer instead of number in JSON schemas where applicable {pull}641[641]
+- Set array item types to string {pull}651[651]
```
Comment (beniwohli) [inline CHANGELOG.asciidoc]: can you add "in JSON schemas" similar to your former entry? That makes the changelog easier to read for people who don't have all the context

Comment (felixbarny) [reply CHANGELOG.asciidoc]: done

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -44,6 +44,7 @@ https://github.com/elastic/apm-server/compare/71df0d96445df35afe27f38bcf734a0828
 - Use updated stack trace frame values for calculating error `grouping_keys` {pull}485[485]
 - Fix panic when a signal is delivered before the server is instantiated {pull}580[580]
 - Use type integer instead of number in JSON schemas where applicable {pull}641[641]
+- Set array item types to string in JSON schemas {pull}651[651]
```
Comment (roncohen) [inline CHANGELOG.asciidoc]: This should not be under 6.2 :)

Comment (simitt) [reply CHANGELOG.asciidoc]: Same for `Use type integer instead of number in JSON schemas where applicable {pull}641[641]`

Comment (felixbarny) [reply CHANGELOG.asciidoc]: woops 😬

**Pull request merged**